android/src/main/java/com/reactnativeone/OneModule.java

```diff
@@ -1,47 +1,59 @@
+import com.facebook.react.bridge.Promise;
+
 import com.facebook.react.bridge.ReadableArray;
 import com.facebook.react.bridge.ReadableMap;
 import com.facebook.react.bridge.ReadableMapKeySetIterator;
 import com.facebook.react.bridge.ReadableNativeMap;
 import com.facebook.react.bridge.WritableMap;
-import com.thunderhead.One;
-import com.thunderhead.OneLogLevel;
-import com.thunderhead.OneModes;
-import com.thunderhead.android.api.codeless.OneCodelessInteractionTrackingConfiguration;
-import com.thunderhead.android.api.configuration.OneConfiguration;
-import com.thunderhead.android.api.interactions.OneCall;
-import com.thunderhead.android.api.interactions.OneCallback;
-import com.thunderhead.android.api.interactions.OneInteractionPath;
-import com.thunderhead.android.api.interactions.OneRequest;
-import com.thunderhead.android.api.interactions.OneResponseCode;
-import com.thunderhead.android.api.interactions.OneResponseCodeRequest;
-import com.thunderhead.android.api.responsetypes.OneAPIError;
-import com.thunderhead.android.api.responsetypes.OneResponse;
-import com.thunderhead.android.api.responsetypes.OneSDKError;
-
-import org.jetbrains.annotations.NotNull;
+import com.facebook.react.bridge.WritableNativeArray;
+import com.facebook.react.bridge.WritableNativeMap;
+import com.thunderhead.mobile.One;
+import com.thunderhead.mobile.codeless.OneCodelessInteractionTrackingConfiguration;
+import com.thunderhead.mobile.configuration.OneConfiguration;
+import com.thunderhead.mobile.configuration.OneMode;
+import com.thunderhead.mobile.interactions.OneInteractionPath;
+import com.thunderhead.mobile.interactions.OneRequest;
+import com.thunderhead.mobile.interactions.OneResponseCode;
+import com.thunderhead.mobile.interactions.OneResponseCodeRequest;
+import com.thunderhead.mobile.logging.OneLogComponent;
+import com.thunderhead.mobile.logging.OneLogLevel;
+import com.thunderhead.mobile.logging.OneLoggingConfiguration;
+import com.thunderhead.mobile.optout.OneOptInOptions;
+import com.thunderhead.mobile.optout.OneOptOutConfiguration;
+
+import com.thunderhead.mobile.responsetypes.OneAPIError;
+import com.thunderhead.mobile.responsetypes.OneResponse;
+import com.thunderhead.mobile.responsetypes.OneSDKError;
+import com.thunderhead.mobile.responsetypes.OptimizationPoint;
 
 import java.net.URI;
 import java.util.ArrayList;
+import java.util.EnumSet;
 import java.util.HashMap;
-import java.util.Map;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.Executors;
+import java.util.concurrent.ExecutorService;
 
 public class OneModule extends ReactContextBaseJavaModule {
 
+  private final ReactApplicationContext reactContext;
+
+  protected static final String LOG_TAG = "OneModule";
+  private final ExecutorService executor = Executors.newSingleThreadExecutor();
+
   public OneModule(ReactApplicationContext reactContext) {
     super(reactContext);
+    this.reactContext = reactContext;
```

Comment:
why do you need this? you can use `getReactApplicationContext` https://github.com/facebook/react-native/blob/main/ReactAndroid/src/main/java/com/facebook/react/bridge/ReactContextBaseJavaModule.java#L35

Reply:
It's not needed and is outdated from library that generates plugin code.  Will remove